bucket/goodbyedpi.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "0.2.1",
+    "description": "Bypass Deep Packet Inspection systems found in many Internet Service Providers which block access to certain websites.",
+    "homepage": "https://github.com/ValdikSS/GoodbyeDPI",
+    "license": "Apache-2.0",
+    "notes": "Follow the instructions on 'https://github.com/ValdikSS/GoodbyeDPI#readme' to set up GoodbyeDPI",
```

Comment:
```suggestion
```

We can omit this because this link is already present in homepage property

Reply:
OK, I'll take your advice.